src/status_im/components/tabs/views.cljs

```diff
@@ -1,32 +1,21 @@
 (ns status-im.components.tabs.views
-  (:require-macros [status-im.utils.views :refer [defview]])
-  (:require [re-frame.core :refer [subscribe dispatch dispatch-sync]]
-            [status-im.components.react :refer [view
-                                                animated-view
-                                                text-input
-                                                text
-                                                image
-                                                touchable-highlight]]
-            [reagent.core :as r]
-            [status-im.components.tabs.styles :as st]
-            [status-im.components.animation :as anim]
-            [status-im.utils.platform :as p]))
+  (:require-macros [status-im.utils.views :refer [defview letsubs]])
+  (:require [re-frame.core :refer [dispatch]]
+            [status-im.components.react :as react]
+            [status-im.components.icons.vector-icons :as vi]
+            [status-im.components.tabs.styles :as styles]))
 
-(defn tab [{:keys [view-id title icon-active icon-inactive selected-view-id]}]
-  (let [active?   (= view-id selected-view-id)]
-    [touchable-highlight {:style    st/tab
-                          :disabled active?
-                          :on-press  #(dispatch [:navigate-to-tab view-id])}
-     [view {:style st/tab-container}
+(defn- tab [{:keys [view-id title icon-active icon-inactive selected-view-id]}]
+  (let [active? (= view-id selected-view-id)]
+    [react/touchable-highlight {:style    styles/tab
+                                :disabled active?
+                                :onPress  #(dispatch [:navigate-to-tab view-id])}
```

Comment:
No more camel-case?

Reply:
do you mean, i should use kebab here? ok

Reply:
Yes